doc/README.md

```diff
@@ -63,14 +72,14 @@ The Bitcoin repo's [root README](/README.md) contains relevant information on th
 - [BIPS](bips.md)
 - [Dnsseed Policy](dnsseed-policy.md)
 - [Benchmarking](benchmarking.md)
-- [Internal Design Docs](design/)
 
 ### Resources
 * Discuss on the [BitcoinTalk](https://bitcointalk.org/) forums, in the [Development & Technical Discussion board](https://bitcointalk.org/index.php?board=6.0).
 * Discuss project-specific development on #bitcoin-core-dev on Libera Chat. If you don't have an IRC client, you can use [web.libera.chat](https://web.libera.chat/#bitcoin-core-dev).
 
 ### Miscellaneous
 - [Assets Attribution](assets-attribution.md)
+- [Assumeutxo design](assumeutxo.md)
```

Comment:
This doesn't exist

Reply:
it's trivial but a sign to be careful about cherry-picking

Reply:
Great catch! This is fixed - the Assumeutxo design now points to the correct file (design/assumeutxo.md)